lightspeed/about/ols-openshift-lightspeed-overview.adoc

```diff
@@ -1,16 +1,14 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="openshift-lightspeed-overview"]
+[id="ols-openshift-lightspeed-overview"]
 include::_attributes/common-attributes.adoc[]
-= OpenShift Lightspeed overview
-:context: openshift-lightspeed-overview
+= {ols} overview
+:context: ols-openshift-lightspeed-overview
 
 toc::[]
 
-{ols-official} provides intelligent, natural language processing capabilities designed to make Red Hat cloud-native application platforms easier to use for novice users and more efficient for experienced professionals. 
+{ols-official} provides intelligent, natural language processing capabilities designed to make Red Hat cloud-native application platforms easier to use for beginners and more efficient for experienced professionals. 
 
 [NOTE]
 ====
-Because {ols} releases on a different cadence from {product-title}, the {ols} documentation is available as a separate documentation set.
-
-The {ols} documentation is available at link:https://docs.openshift.com/lightspeed/1.0tp1/about/ols-about-openshift-lightspeed.html[]
+Because {ols} releases on a different cadence from {product-title}, the {ols} documentation is available as a separate documentation set at link:https://docs.openshift.com/lightspeed/1.0tp1/about/ols-about-openshift-lightspeed.html[About {ols-official}]
```

Comment:
```suggestion
Because {ols} releases on a different cadence from {product-title}, the {ols} documentation is available as a separate documentation set at link:https://docs.openshift.com/lightspeed/1.0tp1/about/ols-about-openshift-lightspeed.html[About {ols-official}].
```